debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/JdbcSinkConnectorConfig.java

```diff
@@ -9,6 +9,8 @@
 import java.util.Set;
 import java.util.function.Consumer;
 
+import io.debezium.connector.jdbc.naming.*;
```

Comment:
We don't permit the use of `.*` imports. 

If you're using IDEA, for example, you can set `Settings > Editor > Code Style > Java > Imports > Class count to use import with '*'` with a value of `999` to avoid the IDE from doing this silly thing.

I'd also run `./mvnw clean install -DskipITs -DskipTests` so that imports get sorted before submitting the PR. We prefer to have packages grouped for legibility reasons.